### DISCUSSION TITLE
Copilot not activated after changing GitHub username

### DISCUSSION THREAD

**Discussion by FranckPoingt** [community/community#41953] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

I changed my GitHub username a few weeks ago. 
Now Copilot doesn't run in my Editor (VS Code), when I check the extension page, under `Runtime Status` it says ** Not yet activated **

How can I troubleshoot that? 

EDIT:
Actually related to [#41947](https://github.com/community/community/discussions/41947)

**Comment by horridharry** [comment]
I understand your error with Copilot may actually be in connection to the recent update which had been pushed out around the early hours of today. Please [refer to this answer](https://github.com/community/community/discussions/41962#discussioncomment-4435896) I made to resolve your issue.
